WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -665,9 +666,17 @@ public void onStart() {
 
     @Override
     public void setTitle(final String title) {
+        if (isAdded()) {
+            mToolbarTitle = title.isEmpty() ? getString(R.string.wordpress_as_site_title) : title;
+
+            if (mToolbar != null) {
+                mToolbar.setTitle(mToolbarTitle);
```

Comment:
Can we refactor the code a bit to avoid having two separate lines in this method where we call `mToolbar.setTitle`? It will make it easier to follow the code if there's only one. Thanks!